app/src/test/java/decide/DecideTest.java

```diff
@@ -45,4 +45,55 @@
         boolean result = Decide.decide(points, params, lcm, puv);
         assertTrue("Simple test case with LIC0 and LIC1", result);
     }
+
+    @Test public void negativeTestCaseWithLIC1and2and6() {
+        //LIC1 is true, 2 and 6 are false
```

Comment:
LIC1 is also false. As you set the RADIUS1 to 100, the 3 points can be contained within the circle of RADIUS1